### PR TITLE
Fix UnhandledRejection

### PR DESCRIPTION
**Pull request by adibas03**

### CODE REVIEW

File: src/TimeNode.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export default class TimeNode {
   public startupMessage(): void {
     this.config.logger.info('EAC-TimeNode');
     this.config.logger.info('Version: ' + Version);
+    this.logNetwork();
```
Comment (Kuzirashi) [inline src/TimeNode.ts]: How that one line prevents errors? : P

Comment (adibas03) [reply src/TimeNode.ts]: Seems, I might have missed some lines.

Comment (adibas03) [reply src/TimeNode.ts]: @Kuzirashi Updated

**Comment by adibas03** [comment]
@Kuzirashi ready!!!

**Pull request merged**